.github/workflows/builds.yml

```
@@ -94,3 +94,46 @@ jobs:
             COMMIT_HASH=${{ github.sha }}
             REVISION=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.revision'] }}
             ENTERPRISE_TOKEN=${{ secrets.ENTERPRISE_TOKEN }}
+  docker-suppression-backup-service:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          context: .
+          file: ./suppression-backup-service/Dockerfile
+          images: |
+            name=rudderlabs/develop-rudder-server-suppression-backup-service
+            name=rudderlabs/rudder-server-suppression-backup-service,enable=${{ github.ref == format('refs/heads/{0}', 'master') || github.event_name == 'release' }}
```

I believe is an overkill to prefix the image name with `rudder-server` even if it is under the same repo.
```suggestion
            name=rudderlabs/develop-suppression-backup-service
            name=rudderlabs/suppression-backup-service,enable=${{ github.ref == format('refs/heads/{0}', 'master') || github.event_name == 'release' }}
```

---

renamed